src/utils/get-experimental-parser-plugins.ts

```diff
@@ -32,3 +32,37 @@ export const getExperimentalParserPlugins = (
         return plugin;
     });
 };
+
+/**
+ * Checks whether a specified plugin is included in importOrderParserPlugins.
+ * More fancy than just a `.includes()` because importOrderParserPlugins can contain plugins with configuration
+ *
+ * @param importOrderParserPlugins array of experimental babel parser plugins
+ * @returns true if the plugin is in the list
+ */
+export const hasPlugin = (
+    importOrderParserPlugins: string[],
+    pluginName: string,
+): boolean => {
+    for (const pluginNameOrJson of importOrderParserPlugins) {
+        const isParserPluginWithOptions = pluginNameOrJson.startsWith('[');
+        let plugin;
+
+        if (isParserPluginWithOptions) {
+            try {
+                plugin = JSON.parse(
```

Comment:
I think you forgot to select the first node in the Tuple `[“pluginName”, …]` needs a `[0]` somewhere near here right?

Reply:
ah, yeah good call